backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImpl.java

```diff
@@ -378,4 +380,9 @@ public void setIncotermLocation(final String incotermLocation)
 		this.incotermLocation = incotermLocation;
 	}
 
+	@Override
+	public int getAD_InputDataSource_ID() {	return AD_InputDataSource_ID;}
+
+	public void setAD_InputDataSource_ID(final int AD_InputDataSource_ID){this.AD_InputDataSource_ID = AD_InputDataSource_ID;}
```

Comment:
instead of this, use lombok (check out e.g. the field `salesRepId`)

Reply:
Why are you not using de.metas.impex.InputDataSourceId ?

Reply:
Also, please, rename this starting with a lowercase letter. That's how we name variables.